### ISSUE TITLE
3D bar graphs fail under RockyLinux 9.3, python3, matplotlib 3.8.2 -  'Axes3D' object has no attribute 'w_xaxis'

### ISSUE THREAD

**Issue by AeonJJohnson** [louwrentius/fio-plot#141]
Attempted below with 'pip3 install' method resulting in fio-plot v1.1.3 and uninstalling and reinstalling v1.1.13 from source. Same result.

fio-plot -i SAMSUNG/nvme1n1/randrw90/4k --source "testing"  -T "Test graph" -L -t lat -r randrw -f read
Traceback (most recent call last):
  File "/usr/local/bin/fio-plot", line 33, in <module>
    sys.exit(load_entry_point('fio-plot==1.1.13', 'console_scripts', 'fio-plot')())
  File "/usr/local/lib/python3.9/site-packages/fio_plot/__init__.py", line 41, in main
    routing_dict[graphtype]["function"](settings, data)
  File "/usr/local/lib/python3.9/site-packages/fio_plot/fiolib/bar3d.py", line 146, in plot_3d
    ax1.w_xaxis.set_ticks(float_x)
AttributeError: 'Axes3D' object has no attribute 'w_xaxis'

**Comment by louwrentius** [comment]
Seems like issue #127 ?

**Comment by cyberguy42** [comment]
@louwrentius I am also encountering this error. It does appear to be a duplicate of #127. 

macOS 12.0.1 with homebrew-installed python: `python --version`
```
Python 3.11.7
```
fio-plot installed into a venv using pip:
```
contourpy==1.2.0
cycler==0.12.1
fio-plot==1.1.14
fonttools==4.48.1
Jinja2==3.1.3
kiwisolver==1.4.5
markdown-it-py==3.0.0
MarkupSafe==2.1.5
matplotlib==3.8.2
mdurl==0.1.2
numpy==1.26.4
packaging==23.2
pillow==10.2.0
pyan3==1.2.0
Pygments==2.17.2
pyparsing==3.1.1
python-dateutil==2.8.2
rich==13.7.0
six==1.16.0
```
Is there any other info that would be helpful?

**Comment by cyberguy42** [comment]
@louwrentius Turned out it was a simple fix, #142 seems to resolve the problem for me

**Comment by louwrentius** [comment]
I've merged #142 so this should be resolved, thanks @cyberguy42
